site/content/docs/main/locations.md

```diff
@@ -166,6 +171,40 @@ During backup creation:
 velero backup create full-cluster-backup
 ```
 
+### Create a storage location that uses different credentials
+
+If your object storage provider plugin supports it, it is possible to create additional `BackupStorageLocations` that use their own credentials.
+It is necessary to follow the instructions for your [object storage provider plugin][5] to install the plugin if not already installed, and create a file with the object storage credentials.
+
+Once you have created the credentials file, create a [Kubernetes Secret][6] in the Velero namespace that contains these credentials:
+
+```shell
+kubectl create secret generic -n velero credentials --from-file=bsl=</path/to/credentialsfile>
+```
+
+This will create a secret named `credentials` with a single key (`bsl`) which contains the contents of your credentials file.
+The name and key of this secret will be given to Velero when creating the `BackupStorageLocation` using the flag `--credential`.
+When interacting with that `BackupStorageLocation`, it will fetch the data from that key within that secret.
+
+For example, a new `BackupStorageLocation` AWS with the created secret would be configured as follows:
+
+```bash
+velero backup-location create <bsl-name> \
+  --provider aws \
+  --bucket <bucket> \
+  --config region=<region> \
+  --credential=credentials=bsl
```

Comment:
Should the `credentials` here match the name of the secret you created above?

Reply:
Yep. I tried to tie this with line 185-186 above, do you think it needs some extra clarification? I found this a bit tricky to explain.

Reply:
I think if its something the user needs to replace, the we should encase it in `<>` like the other inputs or make sure to call out in the paragraph before make sure to say replace `credentials` with the name of your secret